docs/guides/building_on_windows.md

```diff
@@ -0,0 +1,202 @@
+
+To install a build, select "Build" and then "Install dsda-doom". The default directories for build and install are, respectively, `out/build/<config>` and `out/install<config>`.
+
+### Building from the terminal
+
+Make a clone of the repository:
+
+```
+git clone https://github.com/kraflab/dsda-doom.git
+```
+
+Run the CMake configuration:
+
+```
+cd dsda-doom
+cmake -Sprboom2 -Bbuild -DCMAKE_TOOLCHAIN_FILE="C:\vcpkg\scripts\buildsystems\vcpkg.cmake" -DCMAKE_BUILD_TYPE=Release
+```
+
+During this step, vcpkg will build all the dependencies. If vcpkg does not get invoked or CMake fails at finding the dependencies, delete the build directory and make sure the path to the `vcpkg.cmake` toolchain is correct.
+
+And finally, build the project:
+
+```
+cmake --build build --config Release
+```
+
+You should then be able to run dsda-doom from the build directory:
+
+```
+cd build/Release
+./dsda-doom
+```
+
+### Building with Visual Studio Code
+
+Make a clone of the repository:
+
+```
+git clone https://github.com/kraflab/dsda-doom.git
+```
+
+You can either use an extension such as CMake Tools or the integrated terminal to compile. The first time you open the directory, you will be prompted to install recommended extensions for syntax highlighting and CMake support.
+
+If you use the CMake Tools extension, you will need to:
+
+- Set your workspace settings to tell CMake Tools the location of the CMakeLists.txt:
+
+```json
+"cmake.sourceDirectory": "${workspaceFolder}/prboom2"
+```
+
+- Create a new kit (preferably a copy of the one you want to use), and add this key to it:
+
+```json
+"toolchainFile": "C:/vcpkg/scripts/buildsystems/vcpkg.cmake"
+```
+
+## Building with MSYS2
+
+At the time of writing, MSYS2 provides seven different environments. This section assumes you are using UCRT64 (`mingw-w64-ucrt-x86_64-`). If you wish to use another one, refer to [the package naming](https://www.msys2.org/docs/package-naming/) and replace the UCRT64 prefix with the one of your choice.
+
+### Installing dependencies
+
+To download and build the repo, you will need the following tools:
+
+```
+pacman -S mingw-w64-ucrt-x86_64-gcc cmake git ninja pkgconf
+```
+
+Additionally, you will need the following external libraries:
+
+```
+pacman -S mingw-w64-ucrt-x86_64-dumb mingw-w64-ucrt-x86_64-fluidsynth mingw-w64-ucrt-x86_64-libmad mingw-w64-ucrt-x86_64-libvorbis mingw-w64-ucrt-x86_64-portmidi mingw-w64-ucrt-x86_64-SDL2 mingw-w64-ucrt-x86_64-SDL2_image mingw-w64-ucrt-x86_64-SDL2_mixer
+```
+
+### Building
+
+Make a clone of the repository:
+
+```
+git clone https://github.com/kraflab/dsda-doom.git
+```
+
+Run the CMake configuration:
+
+```
+cd dsda-doom
+cmake -Sprboom2 -Bbuild -GNinja -DCMAKE_BUILD_TYPE=Release
+```
+
+And finally, build the project:
+
+```
+cmake --build build --config Release
```

Comment:
I think it worth noting that packaging won't work in MSYS2.

Reply:
You're correct, I've looked into using the CMake facilities to get and install the runtime dependencies, but for libraries such as DUMB and LibMad which don't have a CMake config, or Vorbis which CMake config is not shipped by MSYS2 this mechanism fails.

I will just move that section up to the vcpkg one and add a note about copying DLLs for MSYS2.